### PR TITLE
security/acme-client: add support for new DNS API hooks

### LINKED ISSUE #213: security/acme-client: add support for new DNS API hooks
The updated acme.sh (#210) adds support for new DNS APIs, we'll need to add the (GUI/backend) glue on our side too:

* DigitalOcean API
* DNSimple API
* DuckDNS API
* Dyn Managed DNS
* Dynu API
* Hurricane Electric
* Infoblox API
* Name.com API
* NS1.com API
* VSCALE API
* Yandex PDD API

Not including support for Knot, because they depend on their own version of the nsupdate utility: [knsupdate](https://www.knot-dns.cz/docs/2.4/html/man_knsupdate.html). If a user requests support for it, we may still add it later (and add the required [dependency](http://www.freshports.org/dns/knot2/)).

### PR DESCRIPTION
**Pull request by fraenki**
closes #213 

I don't have accounts for any of these new providers, so I can't test it. The GUI and backend is still working, though... :)

### CODE REVIEW

File: security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml [hunk 1]
```diff
@@ -194,6 +194,30 @@
...
+      <label>DNSimple</label>
+      <type>header</type>
+      <style>table_dns table_dns_dnsimple</style>
+      <help></help>
+    </field>
+    <field>
+        <id>validation.dns_dnsimple_token</id>
+        <label>OAuth Token</label>
+        <type>text</type>
+        <help></help>
```
Comment (fabianfrz) [inline security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml]: fill or remove

File: security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml [hunk 2]
```diff
@@ -194,6 +194,30 @@
...
+      <label>DigitalOcean</label>
+      <type>header</type>
+      <style>table_dns table_dns_dgon</style>
+      <help></help>
+    </field>
+    <field>
+        <id>validation.dns_dgon_key</id>
+        <label>API Key</label>
+        <type>text</type>
+        <help></help>
```
Comment (fabianfrz) [inline security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml]: fill or remove

File: security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml [hunk 3]
```diff
@@ -229,6 +253,58 @@
...
     <field>
+        <label>DuckDNS.org</label>
+        <type>header</type>
+        <style>table_dns table_dns_duckdns</style>
+    </field>
+    <field>
+        <id>validation.dns_duckdns_token</id>
+        <label>API Token</label>
+        <type>text</type>
+        <help></help>
```
Comment (fabianfrz) [inline security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml]: same here

**Comment by fraenki** [comment]
@fabianfrz Thanks! I've removed the empty `<help>` tags and adjusted indention.

**Comment by fraenki** [comment]
Thanks again, @fabianfrz!

**Pull request merged**